### ISSUE TITLE
Ability to turn on/off new funds notifications per group

### ISSUE THREAD

**Issue by jessykate** [cobudget/cobudget#110]
- [x] Only send for positive allocations (no "you have received -$18")
- [x] Ability to configure on/off per group

**Comment by jessykate** [comment]
I've emailed Eugene about this since Derek thinks there was a way they managed to turn notifications off for one group in the past.

**Comment by jessykate** [comment]
Some thoughts on how we might want to implement this:

### group-wide setting
if we create a group-wide setting, then it exists separately from the allocations. 
when an admin goes to allocate, either via CSV or individual allocation
we’d probably need to remind them of the current group setting
and either we’d need to support tweaking the setting in real time, 
or the admin would need to go to a different view to change the setting if appropriate.
this would be one setting applied to the whole group. 

tldr: if we support customizing in the flow, then why bother having a state? and if we don’t support that, it seems like bad UI. 

### one-time indicator
alternatively we can use a *one-time indicator*
keeps the spec closer to the action
allows per-user action
but may mean a group admin has to repeatedly specify their preference (default would be `notify`, i assume)

for the notification request, currently this is just called from the allocation controller
we could pass in a notify=true|false to this method
or we could separate it into a separate API call
which decouples the two in what i think is a more appropriate way
since there’s no reason we shouldn’t be able generate an allocation balance notification at an arbitrary time. 

### current leaning --> stateless with separate API call for notification. 
- add param to UI for notification
- parse this in CSV as well (extra column)
- default to TRUE
- parse in UI; after allocation.save(), call allocation.notify()

also-- we could also have a manual button “notify users of their balances” or “notify this user of their balance" that can be called at any time.

**Comment by jessykate** [comment]
nice work. left a couple comments on the PR :)
